tests/augmentations/test_sepia.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import numpy as np
+
+from discolight.annotations import (annotations_to_numpy_array)
+from discolight.augmentations.sepia import Sepia
+
+def test_my_augmentation_sepia():
+    img =  np.array(([
+        [[ 62, 238, 229],
+         [242,  39, 182]],
+        [[167, 153,  41],
+         [130,  63, 162]]]))
+    
+    annotations = []
+    
+    expected_red = np.array([[250, 159], [191, 130]])
```

Comment:
I assume you calculated these figures by hand? Maybe you want to leave a short comment explaining the logic of this test.